accepted/PSR-4-autoloader-examples.md

```diff
@@ -166,11 +166,11 @@ class Psr4AutoloaderClass
     public function loadClass($class)
     {
         // the current namespace prefix
-        $prefix = $class;
+        $prefix = $class = ltrim($class, '\\');
 
         // work backwards through the namespace names of the fully-qualified
         // class name to find a mapped file name
-        while (false !== $pos = strrpos($prefix, '\\')) {
```

Comment:
Is this change correct? What if the chat is at position 0?

Reply:
@samdark @goetas `$prefix = $class = ltrim($class, '\\');`

Reply:
OK. This one would work.